hbase-client/src/main/java/org/apache/hadoop/hbase/filter/PrefixFilter.java

```diff
@@ -110,9 +110,10 @@ public byte[] toByteArray() {
   }
 
   /**
+   * Parse a serialized representation of {@link PrefixFilter}
    * @param pbBytes A pb serialized {@link PrefixFilter} instance
    * @return An instance of {@link PrefixFilter} made from <code>bytes</code>
-   * @throws org.apache.hadoop.hbase.exceptions.DeserializationException
+   * @throws org.apache.hadoop.hbase.exceptions.DeserializationException if an error occurred
```

Comment:
Just use the simple class name is enough.

Reply:
Done and applied to all other *Filter and *Comparator types.